src/main/java/seedu/mark/commons/core/Messages.java

```diff
@@ -11,4 +13,6 @@
     public static final String MESSAGE_BOOKMARKS_LISTED_OVERVIEW = "%1$d bookmarks listed!";
     public static final String MESSAGE_INVALID_REMINDER_DISPLAYED_INDEX = "The reminder index provided is invalid";
     public static final String MESSAGE_FOLDER_NOT_FOUND = "Folder %1$s does not exist";
+    public static final String MESSAGE_READDING_ROOT_FOLDER = "The " + Folder.DEFAULT_FOLDER_NAME
+            + " folder is the root folder for all folders, choose another name";
```

Comment:
Not very important, but I still can't figure out why it's called `MESSAGE_READDING_ROOT_FOLDER`. Reading?
```suggestion
            + " folder is the root folder for all folders. Please choose another name";
```

Reply:
oh it's re-adding